mlflow/__init__.py

```diff
@@ -24,21 +24,22 @@
 
 For a lower level API, see the :py:mod:`mlflow.tracking` module.
 """
-
 from mlflow.version import VERSION as __version__
+from mlflow.utils.logging_utils import _configure_mlflow_loggers
+import mlflow.tracking.fluent
 
 # Filter annoying Cython warnings that serve no good purpose, and so before
 # importing other modules.
 # See: https://github.com/numpy/numpy/pull/432/commits/170ed4e33d6196d7
 import warnings
 warnings.filterwarnings("ignore", message="numpy.dtype size changed")  # noqa: E402
 warnings.filterwarnings("ignore", message="numpy.ufunc size changed")  # noqa: E402
+warnings.filterwarnings("module", category=DeprecationWarning)
```

Comment:
What does this do? Can we include an inline comment explaining what this does?

Reply:
This sets a filter on any warnings matching the category DeprecationWarning such that warning repetitions are restricted to individual modules, i.e. a deprecation warning is issued for every module where a deprecated function is used.
https://docs.python.org/3/library/warnings.html#the-warnings-filter

Reply:
In-line comment is a good idea